doc/sphinx/source/recipes/recipe_perfmetrics.rst

```diff
@@ -138,23 +139,9 @@ References
 Example plots
 -------------
 
-.. centered:: |pic_permetrics1| |pic_permetrics2|
```

Comment:
In my opinion, it is not acceptable to simply delete these example plots. Simply add proper captions instead.